### PR TITLE
updated release notes

### PR DESCRIPTION
**Pull request by kshuOSIsoft**
Updated release notes

### CODE REVIEW

File: content/release-notes/release-notes.md [hunk 1]
```diff
@@ -4,8 +4,7 @@ uid: ReleaseNotes
 
 # Release notes
 
-PI Adapter for MQTT 1.0 <br>
-Adapter framework 1.3
+PI Adapter for MQTT 1.0
```
Comment (TinaUll) [inline content/release-notes/release-notes.md:8]:
```suggestion
PI Adapter for MQTT 1.0
Adapter framework 1.4
```

File: content/release-notes/release-notes.md [hunk 2]
```diff
@@ -15,55 +14,45 @@ PI Adapter for MQTT is capable of collecting health and diagnostics information.
 
 For more information see [PI Adapter for MQTT overview](xref:PIAdapterForMQTTOverview).
 
-## Setup
-
-### Known issues
+## Known issues
```
Comment (TinaUll) [inline content/release-notes/release-notes.md:17]:
```suggestion
### Known issues
```

File: content/release-notes/release-notes.md [hunk 3]
```diff
@@ -15,55 +14,45 @@ PI Adapter for MQTT is capable of collecting health and diagnostics information.
...
 
 There are no known issues at this time.
 
-### System requirements
+## System requirements
```
Comment (TinaUll) [inline content/release-notes/release-notes.md:21]:
```suggestion
### System requirements
```

File: content/release-notes/release-notes.md [hunk 4]
```diff
@@ -15,55 +14,45 @@ PI Adapter for MQTT is capable of collecting health and diagnostics information.
...
 
 Refer to [System requirements](xref:SystemRequirements).
 
-### Installation and upgrade
+## Installation
```
Comment (TinaUll) [inline content/release-notes/release-notes.md:25]:
```suggestion
### Installation and upgrade
```

File: content/release-notes/release-notes.md [hunk 5]
```diff
@@ -15,55 +14,45 @@ PI Adapter for MQTT is capable of collecting health and diagnostics information.
...
 
 Refer to [Install the adapter](xref:InstallTheAdapter).
 
-### Uninstallation
+## Uninstallation
```
Comment (TinaUll) [inline content/release-notes/release-notes.md:29]:
```suggestion
### Uninstallation
```

**Review by TinaUll — CHANGES_REQUESTED** [review]
@kshuOSIsoft, which template did you use for the release notes? Looks like a lot of the formatting has been reverted in your PR.

File: content/release-notes/release-notes.md [hunk 6]
```diff
@@ -15,55 +14,45 @@ PI Adapter for MQTT is capable of collecting health and diagnostics information.
...
-No additional security vulnerabilities are applicable to this release. Sub-components of this release contain known vulnerabilities which are not exploitable in PI Adapter for MQTT. 
-
-The following table lists the known vulnerabilities and their mitigation in this product.
-
- Component | Version | CVE or Reference | CVSS | Mitigation
- --------- | ------- | -----------------| ------ | ----------
- json.Net | 12.0.3 | [Applications that use Newtonsoft.Json might be exposed to DOS vulnerability](https://alephsecurity.com/vulns/aleph-2018004) | 6.8 | No code paths result in json parsing and subsequent serialization resulting in DoS vulnerability.
-
-## Documentation overview
+This section is intended to provide relevant security-related information to guide your installation or upgrade decision. OSIsoft is proactively disclosing aggregate information about the number and severity of PI Adapter for MQTT security vulnerabilities that are fixed in this release.
```
Comment (TinaUll) [inline content/release-notes/release-notes.md:53]: @kshuOSIsoft This section is intended to list any supporting documentation or reference documentation (like EdgeCmd), it's not intended for security related information.

**Comment by kshuOSIsoft** [comment]
> 
> 
> @kshuOSIsoft, which template did you use for the release notes? Looks like a lot of the formatting has been reverted in your PR.

I copied it from https://github.com/osisoft/PI-Adapter-OPC-UA-Docs/blob/v1.2/content/release-notes/release-notes.md

**Comment by kshuOSIsoft** [comment]
Actually I don't know what happened, let me redo the release notes real quick.

**Review by zryska — APPROVED** [review]

**Pull request merged**